.pre-commit-config.yaml

```diff
@@ -99,3 +102,10 @@ repos:
     rev: 0.4.1
     hooks:
       - id: pydoclint
+
+  - repo: https://github.com/python-poetry/poetry
+    rev: "1.8.2"
```

Comment:
I don't think we hardcode the poetry version anywhere. The devcontainer does specify a version but in our local environments, we might all have different versions of poetry with different behaviours. This is not good

Reply:
I found this long living [thread](https://github.com/python-poetry/poetry/issues/3316) about enforcing Poetry versions, still unsolved.
We could specify it in `install_dependencies.sh`, which is what we trigger when we do `make setup-dev`:
`$ curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.8.2 python3 -`

Reply:
The [recommended](https://python-poetry.org/docs/) way of installing poetry these days is using `pipx`.